invenio/modules/accounts/templates/accounts/settings/helpers_base.html

```diff
@@ -21,7 +21,9 @@
 {%- macro panel_start(title, icon="", btn_text='', btn='', btn_icon='', btn_href='', btn_class='btn-default', btn_name='', id="", with_body=True) %}
 {%- block panel_start scoped %}
 <div class="panel-group panel-bot-margin"{% if id %} id="{{id}}"{% endif %}>
+    {%- block panel_wrapper scoped %}
     <div class="panel panel-default">
+    {%- endblock %}
```

Comment:
I did it like this because i wanted to override just the class
panel-default, but if you think it makes more sense to wrap the whole div
in the block, i can change it.